psamm/commands/table.py

```diff
@@ -0,0 +1,179 @@
+
+
+
+from ..command import Command
+import logging
+import xlsxwriter
+
+logger = logging.getLogger(__name__)
+class ExportTableCommand(Command):
+    """Export parts of the model as tab separated tables"""
+
+
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            'export', metavar='export_type', help='reactions, compounds, medium, limits')
+
+    def run(self):
+        model = self._model
+        export_type = self._args.export
+        if export_type == 'reactions':
+            reaction_export(model)
+        if export_type == 'compounds':
+            compound_parser(model)
+        if export_type == 'medium':
+            media_parser(model)
+        if export_type == 'limits':
+            limits_parser(model)
+        if export_type == 'metadata':
+            metadata_parser(model)
+        if export_type == 'xlsx':
+            xlsx_writer(model)
+
+
+def reaction_export(model):
+    property_set = set()
+    for j in model.parse_reactions():
+        for i, k in j._properties.iteritems():
+            property_set.add(i)
+    property_list = []
+    for i in property_set:
+        property_list.append(i)
+    property_list_sorted = sorted(property_list, key=lambda x:(x!='id', x!='equation', x))
+    for i in model.parse_reactions():
+        print('\t'.join(str(i._properties.get(property)) for property in property_list_sorted))
+
+
+def compound_parser(model):
+    compound_set = set()
+    for compound in model.parse_compounds():
+        for i, j in compound._properties.iteritems():
+            compound_set.add(i)
+    compound_list = []
+    for i in compound_set:
+        compound_list.append(i)
+    compound_list_sorted = sorted(compound_list, key=lambda x:(x!='id', x!='name', x))
+    for compound in model.parse_compounds():
+        print('\t'.join(str(compound._properties.get(property)) for property in compound_list_sorted))
+
+
+def media_parser(model):
+    for entry in model.parse_media():
+        print('{}\t{}\t{}\t{}'.format('Compound ID', 'Reaction ID', 'Lower Limit', 'Upper Limit'))
+        for media_comp in entry:
+            if media_comp[2] == None:
+                lower = -1000
```

Comment:
Instead of `1000` I think this should use the value returned from `model.get_default_flux_limit()`.